other/materials_designer/create_cluster_sphere.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
<p>Let's add a notebook for a custom 3D shape cluster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>